app/models/application_qualification.rb

```diff
@@ -74,7 +74,7 @@ def incomplete_other_qualification?
         send(field_name).blank?
       end
     else
-      return true if EXPECTED_OTHER_QUALIFICATION_DATA.any? do |field_name|
+      return true if EXPECTED_OTHER_QUALIFICATION_DATA.reject { |k| k == :grade }.any? do |field_name|
```

Comment:
I'm wondering the entire `incomplete_other_qualification` could be re-written to make it more readable or at the min, extract the if clauses to methods for readability?

e.g.

```
def incomplete_other_qualification?
   (qualification_type == 'non_uk' && incomplete_international_data?) || incomplete_other_data?
end

```

Reply:
This was a great suggestion Toby, thanks! I worked it through step by step and ended up at the same solution. This method is now much, much easier to read.

Reply:
This actually broke some tests and led me to deep a bit deeper. For non UK qualifications `incomplete_other_data?` is true and the way the old way worked was by returning early so it didn't get that far. 

```  def incomplete_other_qualification?
    if qualification_type == 'non_uk'
      incomplete_international_data?
    elsif other?
      incomplete_other_qualification_data?
    else
      false
    end
  end